docs/GlobalAPI.md

```diff
@@ -0,0 +1,332 @@
+---
+id: api
+title: Globals
+layout: docs
+category: API Reference
+permalink: docs/api.html
+next: expect
+---
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
```

Comment:
The `jest` global is missing from this list. You can add a short entry with the same intro text from `JestObjectAPI.md`, then link to that file for the full reference.

Same goes for the `mockFn` global.